imgui_node_editor.cpp

```diff
@@ -902,6 +926,43 @@ ImCubicBezierPoints ed::Link::GetCurve() const
     return result;
 }
 
+ImLinePoints ed::Link::GetPathSameNode() const
+{
+    ImLinePoints result;
+
+    result.P0 = m_Start + ImVec2(0, 10);
```

Comment:
can these hardcoded values be defined in relation to the node's dimensions? (wide & height)